Products/SQLAlchemyDA/version.txt

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
Is there a reason you added this file? Looks like an accident.

Reply:
I am sorry, there is a reason: the version-file seems to be expected by the code. If the version-file is not avaible, SQLAlchemyDA/da.py throws FileNotFoundError (with Zope5)
![filennotfound](https://user-images.githubusercontent.com/29705216/101989699-99584700-3ca2-11eb-9405-56d37eabecc1.gif)